content.md

```diff
@@ -1133,7 +1158,7 @@ We can retrieve all of the records in the table with the `.all` method:
 
 Notice the class of the return value: `Contact::ActiveRecord_Relation`.
 
-<div class="alert alert-danger">
+<div class="alert alert-success">
```

Comment:
Hmm. See this comment: https://github.com/firstdraft/appdev/issues/618#issuecomment-2417973667

My feeling is that we should be reducing the colors / styles of alerts we use, and not adding new ones? We have never before used `alert-success` in any of our lessons. Perhaps make this an `alert-info`? But `alert-danger` is kind of good because (per that comment) in my mind it is a big red flag to stop and read this important message.

Reply:
I agree, but I think if we use it too many times — i.e. more than once per lesson —  people start tuning it out (I do). I want to reserve danger for the most dangerous of notes to skip.

I'll make it info for now and we can audit later.